res/acmephp.conf.dist

```diff
@@ -57,6 +57,25 @@ storage:
         #   # private_key: path/to/or/contents/of/privatekey
         #   # timeout: 30
 
+        #
+        # Install the certificate in an AWS ELB (Classic Load Balancer)
+        #
+        # - action: install_aws_elb
+        #   region: eu-west-1
+        #   loadbalancer: my_elb
+        #   # certificate_prefix: acmephp_
+        #   # cleanup: true
```

Comment:
Could we use `cleanup_old_certificate` instead, to be clearer? Not a blocker for me though.